examples/use_case_search/server_via_dynamiq.py

```diff
@@ -0,0 +1,77 @@
+import os
+import re
+
+import requests
+
+from dynamiq.utils.logger import logger
+
+ENDPOINT_REPHRASE = "https://cce4b55a-bd05-46c5-a512-a601b092ff82.apps.sandbox.getdynamiq.ai"  # for example
```

Comment:
if someone decides to run it these URL will not be available

Reply:
I will move those URLs to the comments section and add a note that we need to deploy them on the UI first.